src/components/feedbackpage/FeedbackPage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { Button } from 'react-bootstrap';
+
+import { HeaderDashboard } from '../dashboard/HeaderDashboard';
+import { darkModeState } from '../../store/DarkModeState';
+
+import './feedbackpage.scss';
+
+export const FeedbackPage = ()=>{
```

Comment:
Add a space between the parentheses and the arrow function, i.e., change `()=>{` to `() => {`